### PR TITLE
feat: add RecipeRadar crawler

### PR DESCRIPTION
**Pull request by jayaddison**
Adds both user-agent and device matchers for [RecipeRadar](https://www.reciperadar.com/)'s web crawler.

### CODE REVIEW

File: regexes.yaml [hunk 1]
```diff
@@ -179,6 +179,10 @@ user_agent_parsers:
   - regex: '\[FB.{0,300};'
     family_replacement: 'Facebook'
 
+  # RecipeRadar crawler
+  - regex: '(RecipeRadar)/(\d+)\.(\d+)\.(\d+)'
+  - regex: '(RecipeRadar)/(\d+)\.(\d+)'
```
Comment (lbarthon) [inline regexes.yaml:184]: please make the last part optional in the above, saves us an extra regex!

**Comment by jayaddison** [comment]
Thanks for the review @lbarthon - it took me a few moments to understand the `or empty` logic represented by the pipe symbol at the end of group (as used in various other matching regexes), but that makes sense.  Do you have any further feedback on this approach?

**Comment by jayaddison** [comment]
(not quite as long as it took me to remember where I recognized the children's book character in your profile picture from, but some time anyway)

**Review by lbarthon — APPROVED** [review]

**Comment by jayaddison** [comment]
Thank you @lbarthon!

**Pull request merged**